python/cuml/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -873,24 +865,6 @@ class RandomForestClassifier(BaseRandomForestModel,
             return get_rf_json(rf_forest64).decode('utf-8')
         return get_rf_json(rf_forest).decode('utf-8')
 
-    def cpu_to_gpu(self):
```

Comment:
This hack and surrounding code is no longer needed - we add `treelite` to the accelerator's denylist so treelite only ever sees the real `sklearn`.